src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -31,32 +32,64 @@
  * </ul>
  *
  */
-public final class RMSMappingQuality extends InfoFieldAnnotation implements StandardAnnotation {
+public final class RMSMappingQuality extends InfoFieldAnnotation implements StandardAnnotation, ReducibleAnnotation {
```

Comment:
Can you explain why `RMSMappingQuality` implements `ReducibleAnnotation` now and has such a large diff? Don't we have `AS_RMSMappingQuality` (which doesn't extend `RMSMappingQuality`)?